src/main_thread/init/multi_thread_content_initializer.ts

```diff
@@ -1175,6 +1176,22 @@ export default class MultiThreadContentInitializer extends ContentInitializer {
     }
     log.debug("MTCI: Creating ContentDecryptor");
     const contentDecryptor = new ContentDecryptor(mediaElement, keySystems);
+    const updateCodecSupportedByCDM = (state: ContentDecryptorState) => {
+      if (state > ContentDecryptorState.Initializing) {
+        const codecsSupportedByCDM = contentDecryptor.getSupportedCodecs();
+        // TO DO: this needs to be re-done if the keySystem changes.
```

Comment:
I think the behavior is fine here because changing the current keySystem will re-create the content decryptor

Reply:
I have removed the comments